docs/user/whats-new.asciidoc

```diff
@@ -11,4 +12,91 @@ coming[8.0.0]
 
 // tag::notable-highlights[]
 
+[float]
+=== Security on-by-default
+
+Authentication, authorization, and multi-tenancy
+with {kib} Spaces are on-by-default, with almost no configuration effort.
+When you run {kib} for the first time, you'll be presented with a new interactive
+UI that will help you set up {kib} and connect it securely to {es}.
+Simply paste the enrollment token that you received when you ran
+{es}, and {kib} will set up all the previous security features
```

Comment:
```suggestion
{es}, and {kib} will set up all {stack} security features
```

Reply:
I think that referring to the Elastic Stack security features is meaningful here, which is how we discuss them in the Elasticsearch docs. Also, "previous security features" makes it sound like we're including something older, which might make users ask "What about current features?"